### PR TITLE
feat: Initial microsynth of PhishingProtection V1beta1

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-phishing_protection-v1beta1/lib/google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      module V1beta1
+        ##
+        # Service to report phishing URIs.
+        #
+        # To load this service and instantiate a client:
+        #
+        #     require "google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1"
+        #     client = Google::Cloud::PhishingProtection::V1beta1::PhishingProtectionServiceV1Beta1::Client.new
+        #
+        module PhishingProtectionServiceV1Beta1
```
Comment (quartzmo) [inline google-cloud-phishing_protection-v1beta1/lib/google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1.rb:41]: The name of this service (`PhishingProtectionServiceV1Beta1`) doesn't really follow the typical convention. Shouldn't it be just `PhishingProtectionService`? Is this worth investigating upstream?

Comment (dazuma) [reply google-cloud-phishing_protection-v1beta1/lib/google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1.rb:41]: Thanks for catching that. The service name in the upstream proto is indeed `PhishingProtectionServiceV1Beta1` which is terrible, but cannot be changed. I'll look into mapping our class names to something a bit better.

Comment (dazuma) [reply google-cloud-phishing_protection-v1beta1/lib/google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1.rb]: phixed.

**Comment by dazuma** [comment]
This is unblocked.

**Pull request merged**